SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -106,10 +119,10 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
     }
 
     override fun setDepartureStation(station: Station?) {
-        departureStation = station
+        stationRepository.setDepartureStation(station)
     }
```

Comment:
this function is a literal synonym for stationRepository.setDepartureStation - is there some nicer kotlin-y syntax for this type of thing?

Reply:
Not that I know of unfortunately